tests/CustomerDirectDebitValidationTest.php

```diff
@@ -227,19 +227,19 @@ public function testUmlautConversion($schema)
         $xpathDoc->registerNamespace('sepa', 'urn:iso:std:iso:20022:tech:xsd:' . $schema);
         // Date is correctly coded
         $testNode = $xpathDoc->query('//sepa:InitgPty/sepa:Nm');
-        $this->assertEquals('Only A-Z without aeoeuessAeOeUe initiatingPartyName', $testNode->item(0)->textContent);
+        $this->assertEquals('Only A-Z without aeoeueSSAEOEUE initiatingPartyName', $testNode->item(0)->textContent);
```

Comment:
ß is never capitalized into SS, it's always ss and the replacement for the others is always Upper / Lowercase like Oe not OE.